flink-dist/src/test/bin/runBashJavaUtilsCmd.sh

```diff
@@ -34,9 +34,10 @@ bin=`cd "$bin"; pwd`
 
 FLINK_CONF_DIR=${bin}/../../main/resources
 FLINK_TARGET_DIR=${bin}/../../../target
-FLINK_DIST_JAR=`find $FLINK_TARGET_DIR -name 'flink-dist*.jar'`
+FLINK_DIST_JARS=`find ${FLINK_TARGET_DIR} -name 'flink-dist*.jar' -maxdepth 1`
+FLINK_DIST_CLASSPATH=`echo ${FLINK_DIST_JARS[@]} | tr ' ' ':'`
```

Comment:
This works on my Linux machine, but not on my Mac.

The problem is that, `FLINK_DIST_JARS` is actually a string, with jar files separated by line brakes. You can verify this with `echo "$FLINK_DIST_JARS"`.

On my Linux machine, the line brakes are replaced by spaces in the output of `echo ${FLINK_DIST_JARS[@]}`, while preserved on my Mac. Therefore, the `tr` command which tries to replace the spaces with colons does not work for it.

Reply:
I think in order to get it work, we should make sure `FLINK_DIST_JARS` is an array. This could be achieved by simply surrounding the `find` command with `()`.
```
FLINK_DIST_JARS=(`find xxx`)
```